encord/orm/dataset.py

```diff
@@ -793,8 +793,8 @@ def get_str(self) -> str:
             return "GCP_STR"
         elif self == StorageLocation.AZURE:
             return "AZURE_STR"
-        elif self == StorageLocation.OTC:
-            return "OTC_STR"
+        elif self == StorageLocation.S3_COMPATIBLE:
```

Comment:
Oooh. This is dangerous! Have we covered the places where "OTC_STR" is being used? We're potentially going to have a long tail of clients on older versions

Reply:
Discussed this with Justin and Alexey, given that only Floy is using this integration, it should only break it for Floy. And we've sent them an email telling them about this change.